Formula/llvm@6.rb

```diff
@@ -109,6 +109,11 @@ def install
       -DLLVM_CREATE_XCODE_TOOLCHAIN=ON
     ]
 
+    if MacOS.version >= :mojave
+      sdk_path = MacOS::CLT.installed? ? "/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk" : MacOS.sdk_path
```

Comment:
This seems like it should be updated in homebrew/brew itself rather than this formula. What makes llvm@6 need this as a special case vs other formulae?

Reply:
Not sure many other formulae would need such logic unless they are compilers. We need an explicit SDK path so the compiler that is built by the formula (versus the compiler that is building the formula) knows where to find the SDK in future. But, yes, it would be useful logic (if `MacOS::CLT.installed? == False` is truly a supported configuration).